paper/main.tex

```diff
@@ -85,100 +92,142 @@ \section{Background}
-forcing a linear relationship between the payoffs ZDs can ensure that they will
-never receive less than their opponents. The
-American Mathematical Society's news section stated that ``the world of game
-theory is currently on fire''.
-ZDs are indeed a set of mathematically unique strategies
-and robust in pairwise interactions, however, their simplicity and extortionate
-behaviour have been tested. In~\cite{Harper2017} a tournament containing over
-200 strategies, including ZDs, was ran and none of them
-ranked in top spots. Instead, the top ranked strategies were a set of
-trained strategies based on lookup tables~\cite{Axelrod1987}, hidden markov
-models~\cite{Harper2017} and finite state automata~\cite{Miller1996}.
-
-Though only a select pieces of work have been discussed, the IPD literature is
-rich, and new strategies and competitions are being published every year~\cite{Glynatsi2019}. The
-question, however, still remains the same: what is the best way to play the
-game? Compared to other works, whereas a few selected strategies are evaluated
-on a small number of tournaments, this manuscript evaluates the performance of \numberofstrategies
-strategies in \numberofalltournaments tournaments. These tournaments do not
-consist by just standard round robin tournaments, but also by tournaments with noise
-and tournaments with a probabilistic ending. The later part of the paper, evaluates
+robust in tournaments with noise, were soon introduced, including
+Nice and Forgiving~\cite{Bendor1991}, Generous Tit For Tat~\cite{Nowak1992},
+and Pavlov (aka Win Stay Lose Shift)~\cite{Nowak1993}, as well as later
+variants such as OmegaTFT \cite{kendall2007iterated}. Introduction of new strategies
+is often accompanied by a claim that the new strategy is the best known despite
+only being tested against a small number of opponents, or specific classes of
+opponents not necessarily representative of all possible or all published strategies.
+The lack of testing against formally defined strategies and tournament winners
+is understandable given the effort required to implement
+\footnote{Implementing prior strategies faithfully is often extremely difficult or impossible
+due to insufficient descriptions and lack of published implementations or code.}
+the hundreds of published IPD strategies, yet calls into question any claims of superiority
+or robustness of newly introduced
+strategies.
+
+A set of envious IPD strategies were introduced called zero-determinant strategies (ZDs)
+in~\cite{Press2012}. By
+forcing a linear relationship between stationary payoffs ZDs can ensure that they will
+never receive less than their memory-one opponents. While ZDs
+were introduced with a small tournament in which some were reportedly
+successful \cite{Stewart2012}, this result has not generally held in future
+work. In~\cite{Harper2017} a
+series of clever strategies were introduced, and a tournament containing over
```

Comment:
```suggestion
series of strategies trained using reinforcement learning were introduced, and a tournament containing over
```

I'd suggest this is nicer given the way we describe `clever` in the next sentence.